healvis/cosmology.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import absolute_import, division, print_function
+
+import os
+import sys
+import numpy as np
+from astropy.cosmology import Planck15 as cosmo
+from astropy.constants import c
+
+# -----------------------
+#  Functions and constants related
+#  to conversions between instrument and
+#  cosmological units.
+# -----------------------
+
+f21 = 1.420405751e9
+c_ms = c.to('m/s').value
+pi = np.pi
+
+
+def jy2Tsr(f, bm=1.0, mK=False):
```

Comment:
not sure why this was moved to `cosmology` b/c it doesn't require any notion of a cosmology to work properly

Reply:
True, but this way I don't have to import from astropy.constants in utils. I wasn't sure what to call this file exactly, but I'm thinking of it as "any functions needed to go between sky and instrument units".

Reply:
okay, I'm fine with multiple imports of astropy.constants in separate modules. I think having it in utils makes sense b/c it is an in between kind of function that can be used by different modules. I guess my point is, if we keep `cosmology.py` as the module name then lets move the `jy2Tsr` out of it, as someone looking for this functionality might not look in `cosmology.py` to find it

Reply:
Fair enough. I'll move it back to utils.